CONTRIBUTING.md

```diff
@@ -211,9 +211,9 @@ switch to the development database.
 ## Translating
 
 Translations are managed using the
-[Transifex](https://www.transifex.com/openstreetmap/id-editor/) platform. After
+[Transifex](https://explore.transifex.com/openstreetmap/id-editor/) platform. After
```

Comment:
This first link redirected to the project and not to transifex itself, I believe this was intentional.